src/taipy/core/_orchestrator/_orchestrator_factory.py

```diff
@@ -49,48 +49,50 @@ def _build_dispatcher(cls, force_restart=False) -> Optional[_JobDispatcher]:
         if not cls._orchestrator:
             raise OrchestratorNotBuilt
         if Config.job_config.is_standalone:
-            return cls.__build_standalone_job_dispatcher(force_restart=force_restart)
+            cls.__build_standalone_job_dispatcher(force_restart=force_restart)
         elif Config.job_config.is_development:
-            return cls.__build_development_job_dispatcher()
+            cls.__build_development_job_dispatcher()
+        elif util.find_spec(cls._TAIPY_ENTERPRISE_MODULE) is not None:
```

Comment:
```suggestion
        elif util.find_spec(cls._TAIPY_ENTERPRISE_MODULE):
```
This should be enough right?